pure_protobuf/message.py

```diff
@@ -125,10 +125,10 @@ def dumps(self) -> bytes:
 
     def __setattr__(self, name: str, value: Any) -> None:  # noqa: D105
         super().__setattr__(name, value)
-        descriptor = self.__PROTOBUF_FIELDS_BY_NAME__[name]
-        one_of = descriptor.one_of
-        if one_of is not None:
-            one_of._keep_attribute(self, descriptor.number)
+        if descriptor := (value is not None and self.__PROTOBUF_FIELDS_BY_NAME__.get(name)):
```

Comment:
Thanks for the submission!

There's one major concern here: `__PROTOBUF_FIELDS_BY_NAME__` must always contain a descriptor for the `name`, so calling `.get()` here might potentially mask a bug.

Also, the `value is not None` should've been outside the descriptor assignment.

Not to worry, I'll make a new PR